scripts/triggerShortcut.ts

```diff
@@ -134,7 +134,9 @@ void (async () => {
   }
 
   const account = mnemonicToAccount(argv.mnemonic, {
-    path: argv.derivationPath as any,
+    ...(argv.networkId === NetworkId['celo-mainnet'] && {
+      path: argv.derivationPath as any,
+    }),
```

Comment:
I'd expect the derivation path to be used if passed from argv.
It seems what we'd want is change the default derivation path used (when not provided) when the network is Celo vs others. (so it behaves like our wallet).